pkgs/by-name/is/iscc/package.nix

```
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchurl
+, innoextract
+, runtimeShell
+, wineWow64Packages
+, lib
+}:
+
+let
+  version = "6.2.2";
+  majorVersion = builtins.substring 0 1 version;
+in
+stdenv.mkDerivation rec {
+  name = "iscc";
```

```suggestion
  pname = "iscc";
```

`name` will be populated by stdenv from the `pname` and `version` and whatever prefixes are needed